afl-system-config

```diff
@@ -76,6 +76,9 @@ EOF
   DONE=1
 fi
 if [ "$PLATFORM" = "OpenBSD" ] ; then
+  doas sysctl vm.malloc_conf=
+  echo 'Freecheck on allocation in particular can be detrimental to performance.'
+  echo 'Also we might not necessarily to abort at any allocation failure.'
```

Comment:
looks good, just fix the grammar of this sentence please :)

Reply:
:)